packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -106,103 +108,115 @@ const ProfilePanel: FC = () => {
-                    disabled={isLoading}
-                    {...form.getInputProps('firstName')}
-                />
-
-                <TextInput
-                    placeholder="Last name"
-                    label="Last name"
-                    type="text"
-                    required
-                    disabled={isLoading}
-                    {...form.getInputProps('lastName')}
-                />
-
-                <TextInput
-                    placeholder="Email"
-                    label="Email"
-                    type="email"
-                    required
-                    disabled={isLoading}
-                    inputWrapperOrder={[
-                        'label',
-                        'input',
-                        'error',
-                        'description',
-                    ]}
-                    {...form.getInputProps('email')}
-                    rightSection={
-                        isEmailServerConfigured && data?.isVerified ? (
-                            <Tooltip label="This e-mail has been verified">
+        <MantineProvider theme={getMantine8ThemeOverride()}>
+            <form onSubmit={handleOnSubmit}>
+                <Stack mt="md">
+                    <TextInput
+                        placeholder="First name"
+                        label="First name"
+                        type="text"
+                        required
+                        disabled={isLoading}
+                        {...form.getInputProps('firstName')}
+                    />
+
+                    <TextInput
+                        placeholder="Last name"
+                        label="Last name"
+                        type="text"
+                        required
+                        disabled={isLoading}
+                        {...form.getInputProps('lastName')}
+                    />
+
+                    <TextInput
+                        placeholder="Email"
+                        label="Email"
+                        type="email"
+                        required
+                        disabled={isLoading}
+                        inputWrapperOrder={[
+                            'label',
+                            'input',
+                            'error',
+                            'description',
+                        ]}
+                        {...form.getInputProps('email')}
+                        rightSection={
+                            isEmailServerConfigured && data?.isVerified ? (
+                                <Tooltip label="This e-mail has been verified">
+                                    <MantineIcon
+                                        size="lg"
+                                        icon={IconCircleCheck}
+                                        // TODO: Move inline styles to theme extension
+                                        style={{
+                                            color: 'var(--mantine-color-green-6)',
```

Comment:
this is interesting because it could just use `color='green.6'` but this is fine